split.Console/Models/Blackjack/BlackjackCard.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackCard
+    {
+        public SuitType Suit;
+        public int Rank;        //TODO Decide if this would be better as an enum
+
+        public BlackjackCard(SuitType suit, int rank)
```

Comment:
This was meant to be moved but I guess I never did. It will be a level up and just Card.cs next commit